ql/pom.xml

```diff
@@ -437,6 +437,29 @@
       <artifactId>opencsv</artifactId>
       <version>${opencsv.version}</version>
     </dependency>
+    <dependency>
```

Comment:
I was already doubting... You are absolutely right. I am going to rewrite the parts that are dependent in order to be able to remove this. Maybe I have to bring some constants and create them in Hive, but I still need think it makes sense to bring them instead of bringing the full dependencies.